cse/reader/CommentReader.py

```diff
@@ -145,7 +144,10 @@ def __next__(self):
             raise stop
 
         iterRow = next(csv.reader([line], delimiter=self.__delimiter))
-        return self.__parseIterRow(iterRow, skipArticleMapping=False)
+        try:
+            return self.__parseIterRow(iterRow, skipArticleMapping=False)
+        except ValueError:
+            return self.__next__()
```

Comment:
I think, this error handling can be kept in the code base

Reply:
ok